apps/system/test/unit/statusbar_test.js

```diff
@@ -797,6 +797,16 @@ suite('system/Statusbar', function() {
       assert.isTrue(setAppearanceStub.calledWith(app));
     });
 
+    test('rocketbar-deactivated', function() {
+      StatusBar.element.classList.add('hidden');
+      assert.isTrue(StatusBar.element.classList.contains('hidden'));
```

Comment:
To be honest, just copied and pasted the test for 'stackchanged' event, as we should do the same... but you are right, it doesn't make any sense. I'll remove both in order to clean it up. Thanks!